README.md

```diff
@@ -178,3 +178,52 @@ Part 2: Update the build settings
         GoTo General Settings in Travis and enable only the "Build Push Requests". Disable "Build Pushed Branches".
     Step 2: Enable auto cancelling builds
         GoTo Setting in Travis. Under Auto Cancellation enable "Auto cancel branch builds".
+
+Hosting the Todo App on Heroku
```

Comment:
You can preview markdown (this is a .md file) in VSCode via a button in the top right, or you can view it in GitHub.

Markdown has syntax for styling it which you haven't taken advantage of but maybe want to. And there some important points to remember like the fact that line breaks don't count unless you have a blank line or e.g. a list marker (`-` or `*`)

https://github.com/github-virendra/-DevOps-Course-Starter/blob/81871555b628faa32e48963ae00b8ae05e9fa37a/README.md